### PR TITLE
test(rust): move bats tests artifacts to a non-temporary directory to keep them in case of failure

### PR DESCRIPTION
**Pull request by adrianbenavides**

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/tests/bats/load/base.bash [hunk 1]
```diff
@@ -65,7 +65,14 @@ setup_home_dir() {
   export OCKAM_HOME="$dir"
 }
 
+mkdir -p "$HOME/.bats-tests"
 teardown_home_dir() {
+  # If BATS_TEST_COMPLETED is not set, the test failed.
+  if [[ -z "$BATS_TEST_COMPLETED" ]]; then
+    # Copy the CLI directory to $HOME/.bats-tests so it can be inspected.
+    # For some reason, if the directory is moved, the teardown function gets stuck.
+    cp -a "$OCKAM_HOME" "$HOME/.bats-tests"
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_command/tests/bats/load/base.bash:74]: Would this do the trick on CI? @metaclips

**Review by etorreborre — APPROVED** [review]
Let's wait for @metaclips validation but that looks good to me. 
Question: how do we access those files once they have been created on CI?

File: implementations/rust/ockam/ockam_command/tests/bats/load/base.bash (hunk 1 above)
Comment (metaclips) [reply implementations/rust/ockam/ockam_command/tests/bats/load/base.bash:74]: It should do, thanks.

**Comment by metaclips** [comment]
> Let's wait for @metaclips validation but that looks good to me. Question: how do we access those files once they have been created on CI?

We could use GitHub artifacts https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts

**Pull request merged**